aodntools/timeseries_products/velocity_hourly_timeseries_template.json

```
@@ -0,0 +1,218 @@
+            "standard_name": "northward_sea_water_velocity",
+            "units": "m s-1",
+            "valid_max": 10.0,
+            "valid_min": -10.0
+            },
+        "VCUR_max": {
+            "units": "m s-1",
+            "standard_name": "northward_sea_water_velocity",
+            "long_name": "max data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:max"
+            },
+        "VCUR_min": {
+            "units": "m s-1",
+            "standard_name": "northward_sea_water_velocity",
+            "long_name": "min data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:min"
+            },
+        "VCUR_std": {
+            "units": "m s-1",
+            "standard_name": "northward_sea_water_velocity",
+            "long_name": "std data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:std"
+            },
+        "VCUR_count": {
+            "standard_name": "northward_sea_water_velocity number_of_observations",
+            "units": "1",
+            "long_name": "count data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:count"
+            },
+
+        "WCUR": {
+            "coordinates": "TIME DEPTH LATITUDE LONGITUDE",
+            "ancillary_variables": "WCUR_max, WCUR_min, WCUR_std, WCUR_count",
+            "long_name": "upward_sea_water_velocity",
+            "standard_name": "upward_sea_water_velocity",
+            "units": "m s-1",
+            "valid_max": 5.0,
+            "valid_min": -5.0
+            },
+        "WCUR_max": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "max data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:max"
+            },
+        "WCUR_min": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "min data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:min"
+            },
+        "WCUR_std": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "std data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:std"
+            },
+        "WCUR_count": {
+            "standard_name": "upward_sea_water_velocity number_of_observations",
+            "units": "1",
+            "long_name": "count data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:count"
+            },
+        "CELL_INDEX": {
+            "long_name": "index of the corresponding measuring cell",
+            "comment": "the closest cell to the sensor is cell 0"
+        },
+        "SECONDS_TO_MIDDLE": {
+            "long_name": "number of seconds from timestamp to middle of the measurement window"
```

Do you think we need this variable in the hourly product?
If we do leave it in, I think this would be clearer:
```suggestion
            "long_name": "offset in seconds from timestamp to middle of the measurement window in the input file"
```